publish.sh

```diff
@@ -72,6 +72,10 @@ npm install
 npm run build
 popd
 
+pushd pca-ui/src/witch
+npm run build
+popd
+
```

Comment:
Should also remove the code that downloads the witch file, adds to artifacts bucket, and the 'sed' code that inline edits WITCHKEY token

Reply:
Done